.github/workflows/dev_on_workflow_environment_ios_deploy.yml

```
@@ -91,7 +92,11 @@ jobs:
           ssh-private-key: ${{  steps.secrets.outputs.MATCH_SSH_KEY }}
       - name: Deploy hard ios for ${{ inputs.ENV }}
         run: |
-          bundle exec fastlane ios deploy --env ${{ inputs.ENV }} --verbose
+          if [ "${{ inputs.ENV }}" != "production" ]; then
+            bundle exec fastlane ios deploy firebase_token:${{ steps.secrets.outputs.FIREBASE_TOKEN }} --env ${{ inputs.ENV }} --verbose
+          else
+            bundle exec fastlane ios deploy --env ${{ inputs.ENV }} --verbose
+          fi
```

je trouve ça plus facile de lire des conditions positives (sans négations)

```suggestion
          if [ "${{ inputs.ENV }}" == "production" ]; then
            bundle exec fastlane ios deploy --env ${{ inputs.ENV }} --verbose
          else
            bundle exec fastlane ios deploy firebase_token:${{ steps.secrets.outputs.FIREBASE_TOKEN }} --env ${{ inputs.ENV }} --verbose
          fi
```